ostd/src/arch/x86/iommu/registers/mod.rs

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Registers and their definition used by IOMMU.
+
+mod capability;
+mod command;
+mod extended_cap;
+mod status;
+
+use bit_field::BitField;
+pub use capability::*;
+use command::GlobalCommand;
+#[allow(unused_imports)]
+pub use command::*;
+pub use extended_cap::*;
```

Comment:
Do not write `pub use submodule::*`, which is an anti-pattern that harms code readability.